### ISSUE TITLE
Error: File outside of allowed directories

### ISSUE THREAD

**Issue by ewingrj** [embark-framework/embark-solc#20]
I'm not sure why this is throwing an error, b/c other contract in `/node_modules/giveth-liquidpledging/contracts` seem to be allowed to import.

```
/Users/rjewing/code/giveth/lpp-campaign/node_modules/giveth-liquidpledging/contracts/lib/aragon/IKernelEnhanced.sol:3:1: ParserError: Source "@aragon/os/contracts/lib/misc/ERCProxy.sol" not found: File outside of allowed directories.
import "@aragon/os/contracts/lib/misc/ERCProxy.sol";
^--------------------------------------------------^
```

Here is the repo & commit this happened in https://github.com/Giveth/lpp-campaign/commit/61f152ba585525e466afa54e50ba92b91323576d.

You'll need to enable the `embark-solc` plugin in `embark.json`

**Comment by jrainville** [comment]
Thanks fir reporting. Just added it to our tracking.
We'll probably fix it next week or the week after.

**Comment by emizzle** [comment]
@perissology - 
This will be fixed when the following PRs are merged:
1. Fix race condition in `embark-solc` - https://github.com/embark-framework/embark-solc/pull/21
2. Add recursive contract imports in `embark` - https://github.com/embark-framework/embark/pull/1260

**Comment by jrainville** [comment]
Fixed and published in 4.0.0
